docs/pages/build-reference/variables.md

```diff
@@ -61,9 +61,108 @@ See the [eas.json reference](/build/eas-json.md) for more information.
+
+![App secrets location](/static/images/eas-build/environment-secrets/secrets-project-nav.png)
+
+If you haven't published your app yet and it isn't visible on the projects list, use the create a project on the project list page.
+
+![Create project button location](/static/images/eas-build/environment-secrets/project-creation-navigation.png)
+
+![Create project UI](/static/images/eas-build/environment-secrets/project-creation-web.png)
+
+### Adding secrets from the web UI
+
+When setting up secrets for a new account or app, you'll be met with this UI:
+
+![Empty secrets UI](/static/images/eas-build/environment-secrets/secrets-empty.png)
+
+Click the "Create" button in the top-right of the table to create a new secret.
+
+A secret needs a name and a value. The name can only contain alphanumeric characters and underscores:
+
+![Secret creation UI filled](/static/images/eas-build/environment-secrets/secrets-create-filled.png)
+
+![Secret UI with stored secret](/static/images/eas-build/environment-secrets/secrets-populated.png)
+
+### Adding secrets from EAS CLI
+
+To create a new secret, run `eas secrets:create`
+
+```
+> eas secrets:create project SECRET_NAME secretvalue
+✔ Linked to project @fiberjw/keitai-demo-2
+✔ You're inside the project directory. Would you like to use fiberjw account? … yes
+✔ ️Created a new secret SECRET_NAME on project @fiberjw/keitai-demo-2.
+```
+
+To view the secrets you currently have for this project, run `eas secrets:list`:
+
+```
+> eas secrets:list
+✔ Linked to project @fiberjw/keitai-demo-2
+Secrets for this account and project:
+┌─────────────────┬─────────┬─────────────────┬──────────────────────────────────────┐
+│ Name            │ Target  │ Updated at      │ ID                                   │
+├─────────────────┼─────────┼─────────────────┼──────────────────────────────────────┤
+│ Anotherone      │ project │ Mar 11 17:51:36 │ e6625438-d1ed-463b-a143-dd3c2d8f57d6 │
+├─────────────────┼─────────┼─────────────────┼──────────────────────────────────────┤
+│ sdalskdasdlkasj │ project │ Mar 14 20:57:31 │ f093af84-cc8e-45c0-b969-0e86c724369d │
+├─────────────────┼─────────┼─────────────────┼──────────────────────────────────────┤
+│ hellllo         │ account │ Mar 14 20:10:52 │ aa08a553-289e-4a6a-9063-8607a4358df5 │
+├─────────────────┼─────────┼─────────────────┼──────────────────────────────────────┤
+│ sdalskdasdlkasj │ account │ Mar 14 22:10:06 │ 52904cd8-8b23-4afc-aaba-403b9ded7f9f │
+├─────────────────┼─────────┼─────────────────┼──────────────────────────────────────┤
+│ SECRET_NAME     │ project │ Mar 15 17:46:08 │ 0fdde012-43e2-433d-abb6-9f5b01b63cdf │
+└─────────────────┴─────────┴─────────────────┴──────────────────────────────────────┘
+```
+
+To delete a secret, run `eas secrets:delete`:
+
+```
+> eas secrets:delete 5ba31194-3fc1-45d0-a76c-d5c2ea09c9b9
+
+You are about to permamently delete secret with id: "5ba31194-3fc1-45d0-a76c-d5c2ea09c9b9".
```

Comment:
i know this is more of a cli thing but we should probably output the name of the secret when deleting